hexrd/ui/image_series_toolbar.py

```diff
@@ -93,9 +113,18 @@ def update_omega_label_text(self):
         ome_range = HexrdConfig().omega_ranges
 
         enable = not is_aggregated and ome_range is not None
-        self.omega_label.setVisible(enable)
+        self.toggle_widget_visibility(enable)
```

Comment:
I think we should change this one back. It is possible for an image stack to be loaded with no omega information associated with it. If that's the case, the omega label should be invisible, but the slider and frame number should be visible.